app/views/proposals/detail/_summary.html.haml

```diff
@@ -0,0 +1,24 @@
+
+.card
+  .card-head.column
+    .row
+      .medium-9.column
+        %h3
+          = proposal.public_id
+        %h1.c2n_header
+          = proposal.name
+        .c2n_description
+
+          %p
+            Requested by:
+            %strong.requested-by-name
+              = proposal.requester.full_name
+            Requested on:
+            %strong
+              %span
+                = proposal.created_at.strftime("%b %d, %Y at %l:%M%P")
+      .medium-3.column.text-center
+        %span.status-wrap.medium-12.small-12{ class: "status-is-" + @proposal.status }
+          Status:
+          %span.status-value
+            = @proposal.status.capitalize
```

Comment:
have you considered using `capitalized_detailed_status` from `ProposalDecorator`? That's what we are using in emails.